### PR TITLE
#430 invitations Email Template

### PR DESCRIPTION
**Pull request by Yvan-David**
### PR Description
Please include a brief description of the pull request you want to raise.

- implement styling on invitations email template

### Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you or that you assigned yourself. Make sure you check evey completed task.

1.  implement styling on the invite User template

### Functionality
How does this work, add some information that may be needed in testing
### How has this been tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
### PR Checklist:
- [ ] Task 1.
- [ ] Task 2.
- [ ] Task 3.
- [ ] Task n.
### 430
Trello Link (#DP-?)
### Screenshots (If appropriate)
Link to the video showing how all templates are looking after working on them:
[https://www.loom.com/share/2f7ec4a79d0144ca84adf55ef963d1af?sid=b27d9084-9d7b-45ec-a8ae-a40c93d4e120](https://www.loom.com/share/2f7ec4a79d0144ca84adf55ef963d1af?sid=b27d9084-9d7b-45ec-a8ae-a40c93d4e120)

### CODE REVIEW

**Review by niyobertin — APPROVED** [review]
You can squash the commit to have at least 2 commit messages.

File: .env.example [hunk 1]
```diff
@@ -15,3 +15,9 @@ ACCESS_TOKEN=<Oauth-access_token>
 SENDER_EMAIL=<sender_email>
 OAUTH_TYPE=<Oauth_type>
 OAUTH_HOST=<oauth_host>
+COORDINATOR_EMAIL=<****>
+COORDINATOR_PASS=<****>
+MANAGER_EMAIL=<****>
+MANAGER_PASSWORD=<****>
+ADMIN_EMAIL=<****>
+ADMIN_PASS=<****>
```
Comment (ncutixavier) [inline .env.example]: Why do you need this?

**Review by ncutixavier — COMMENTED** [review]
Make only necessary changes

**Review by YvetteNyibuka — APPROVED** [review]

**Review by Oliviier-dev — APPROVED** [review]
LGTM

File: package.json [hunk 2]
```diff
@@ -77,6 +77,7 @@
     "node-fetch": "^2.6.12",
     "nodemailer": "^6.7.8",
     "normalize-mongoose": "^1.0.0",
+    "npm-force-resolutions": "^0.0.10",
```
Comment (ncutixavier) [inline package.json]: Do we need this package?

File: src/index.ts [hunk 3]
```diff
@@ -70,7 +70,7 @@ export const resolvers = mergeResolvers([
 ]);
 
 async function startApolloServer(typeDefs: DocumentNode, resolvers: IResolvers) {
-  const app = express();
+  const app = express() as any;;
```
Comment (ncutixavier) [inline src/index.ts]: Why do we need casting?

**Review by ncutixavier — CHANGES_REQUESTED** [review]

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**